packages/mwp-app-route-plugin/src/route.js

```diff
@@ -12,6 +12,11 @@ export const onPreResponse = {
 	 */
 	method: (request: HapiRequest, reply: HapiReply) => {
 		const response = request.response;
+
+		if (response && response.header) {
```

Comment:
`response` is guaranteed to be a valid object with a `header` method here - no need to check for a falsey value